app/main/views/create_a_brief/delete.py

```diff
@@ -27,7 +49,11 @@ def delete_a_brief(framework_slug, lot_slug, brief_id):
     if not is_brief_correct(brief, framework_slug, lot_slug, current_user.id) or not brief_can_be_edited(brief):
         abort(404)
 
-    data_api_client.delete_brief(brief_id, current_user.email_address)
-    flash(BRIEF_DELETED_MESSAGE.format(brief=brief), "success")
+    if "delete_confirmed" in request.form:
+        data_api_client.delete_brief(brief_id, current_user.email_address)
+        flash(BRIEF_DELETED_MESSAGE.format(brief=brief), "success")
+        return redirect(url_for(".buyer_dos_requirements"))
```

Comment:
I don't think we need `delete_cofirmed` here or in the button any more, since Cancel is a link